radiolog.py

```diff
@@ -4828,10 +4762,9 @@ def removeTab(self,caller):
 	def autoCleanup(self): # this function is called every second by the timer
 		if self.ui.autoCleanupCheckBox.isChecked():
 			for tab in newEntryWidget.instances:
-				# rprint("lastModAge:"+str(tab.lastModAge))
+# 				rprint("lastModAge:"+str(tab.lastModAge))
 				# note the pause happens in newEntryWidget.updateTimer()
 				if tab.ui.messageField.text()=="" and tab.lastModAge>60:
-					rprint('  closing unused new entry widget for '+str(tab.ui.teamField.text())+' due to inactivity')
```

Comment:
preserve code from #504 - it would be a valuable part of the transcript in the future, and doesn't happen frequently enough to worry about file size